rootfs/ingest/python/qserv/ingest.py

```diff
@@ -305,21 +308,20 @@ def _ingest_all_contributions(self, transaction_id: int, contributions: list[Con
                 loop = False
             else:
                 _LOG.info(
-                    "Contributions for transaction %s, RECENTLY STARTED: %s, NOT FINISHED: %s," +
-                    "RECENTLY FINISHED: %s, FINISHED: %s",
+                    "Contributions for transaction %s, RECENTLY STARTED: %s, NOT FINISHED: %s,"
+                    + "RECENTLY FINISHED: %s, FINISHED: %s",
```

Comment:
You don't need `+` to concatenate literal strings.
Add space before `RECENTLY`.